samples/highcharts/exporting/multiple-charts/demo.js

```diff
@@ -45,7 +45,7 @@ Highcharts.exportCharts = function (charts, options) {
     options = Highcharts.merge(Highcharts.getOptions().exporting, options);
 
     // Post to export server
-    Highcharts.post(options.url, {
+    Highcharts.Chart.prototype.post(options.url, {
```

Comment:
We have to continue with exposing `post` function on the `Highcharts` namespace. This goes for all modified samples. So the `post` function of the `Exporting` module has to be assigned in its masters file to `G.post`.